generated/nirfmxinstr_restricted/nirfmxinstr_restricted.proto

```diff
@@ -0,0 +1,440 @@
+  rpc SetForceAllTracesEnabled(SetForceAllTracesEnabledRequest) returns (SetForceAllTracesEnabledResponse);
+  rpc SaveConfigurationsToJSON(SaveConfigurationsToJSONRequest) returns (SaveConfigurationsToJSONResponse);
+  rpc LoadConfigurationsFromJSON(LoadConfigurationsFromJSONRequest) returns (LoadConfigurationsFromJSONResponse);
+  rpc GetAttributeDesiredF64(GetAttributeDesiredF64Request) returns (GetAttributeDesiredF64Response);
+  rpc GetAttributeDesiredI32(GetAttributeDesiredI32Request) returns (GetAttributeDesiredI32Response);
+  rpc GetAttributeDesiredI64(GetAttributeDesiredI64Request) returns (GetAttributeDesiredI64Response);
+  rpc GetAttributeDesiredString(GetAttributeDesiredStringRequest) returns (GetAttributeDesiredStringResponse);
+  rpc GetAttributeDesiredF64Array(GetAttributeDesiredF64ArrayRequest) returns (GetAttributeDesiredF64ArrayResponse);
+  rpc GetAttributeDesiredF32Array(GetAttributeDesiredF32ArrayRequest) returns (GetAttributeDesiredF32ArrayResponse);
+  rpc GetAttributeDesiredF32(GetAttributeDesiredF32Request) returns (GetAttributeDesiredF32Response);
+  rpc GetAttributeAuthor(GetAttributeAuthorRequest) returns (GetAttributeAuthorResponse);
+  rpc GetPrivilegeLevel(GetPrivilegeLevelRequest) returns (GetPrivilegeLevelResponse);
+  rpc RequestPrivilege(RequestPrivilegeRequest) returns (RequestPrivilegeResponse);
+  rpc SaveAllForRevert(SaveAllForRevertRequest) returns (SaveAllForRevertResponse);
+  rpc LoadAllForRevert(LoadAllForRevertRequest) returns (LoadAllForRevertResponse);
+  rpc GetInitiaitedSnapshotStrings(GetInitiaitedSnapshotStringsRequest) returns (GetInitiaitedSnapshotStringsResponse);
+  rpc GetSnapshotInfoFromCache(GetSnapshotInfoFromCacheRequest) returns (GetSnapshotInfoFromCacheResponse);
+  rpc GetLatestConfigurationSnapshot(GetLatestConfigurationSnapshotRequest) returns (GetLatestConfigurationSnapshotResponse);
+  rpc GetSnapshotState(GetSnapshotStateRequest) returns (GetSnapshotStateResponse);
+  rpc DeleteSnapshot(DeleteSnapshotRequest) returns (DeleteSnapshotResponse);
+  rpc ConvertForPowerUnitsUtility(ConvertForPowerUnitsUtilityRequest) returns (ConvertForPowerUnitsUtilityResponse);
+  rpc UnregisterSpecialClientSnapshotInterest(UnregisterSpecialClientSnapshotInterestRequest) returns (UnregisterSpecialClientSnapshotInterestResponse);
+  rpc GetCalibrationPlaneNames(GetCalibrationPlaneNamesRequest) returns (GetCalibrationPlaneNamesResponse);
+  rpc GetCalibrationPlaneEnabled(GetCalibrationPlaneEnabledRequest) returns (GetCalibrationPlaneEnabledResponse);
+  rpc GetExternalAttenuationTableNames(GetExternalAttenuationTableNamesRequest) returns (GetExternalAttenuationTableNamesResponse);
+  rpc GetActiveTableName(GetActiveTableNameRequest) returns (GetActiveTableNameResponse);
+  rpc GetSParameterExternalAttenuationType(GetSParameterExternalAttenuationTypeRequest) returns (GetSParameterExternalAttenuationTypeResponse);
```

Comment:
Please remove this method - this is from the public header and was added to [nirfmxinstr.proto](https://github.com/ni/grpc-device/blob/main/generated/nirfmxinstr/nirfmxinstr.proto#L71).

Reply:
Done. Thanks for letting me know!